android-sdk/android/src/main/kotlin/io/logto/sdk/android/auth/social/wechat/WechatSocialResultActivity.kt

```diff
@@ -24,7 +24,7 @@ open class WechatSocialResultActivity : Activity(), IWXAPIEventHandler {
             val redirectTo = it.redirectTo
             val appId = Uri.parse(redirectTo).getQueryParameter("app_id")
             if (appId.isNullOrBlank()) {
-                it.handleMissingAppIdError()
+                it.handleMissingInformationError()
```

Comment:
Seems that we should also handle the case of missing `state`?

Reply:
According to our convention, we only report `insufficient_information` error.
The same as Swift SDK.
https://www.notion.so/silverhand/Social-Errors-94f8499b728947fa8309267f69a7d7c5#caafd4ad97d4459d8a84302c923b6a72